frontend/src/metabase/visualizations/echarts/cartesian/option/series.ts

```diff
@@ -184,6 +185,12 @@
             chartModel.yAxisExtents,
             renderingContext,
           );
+        case "waterfall":
```

Comment:
In this PR the waterfall series is returned here, in a later PR it will be moved